### ISSUE TITLE
Loss and val_loss

### ISSUE THREAD

**Issue by lenhatdong** [pierluigiferrari/ssd_keras#243]
Hi,
Thanks for sharing your project.
I trained it on my own data with 20.000 images and 3 classifications.
I tried with 32 epochs and learning rate = 0.0005 ( because lr = 0.001 caused NaN in my loss), i changed lots of parameters however my loss ans val_loss converge always to about 6, NOT 0.
Do you think that this problem come from the small number of epochs or learning rate?
Can you share your loss and val_loss values or their graph, please?

**Comment by buaa-luzhi** [comment]
@mikado3119 
Hello, I have changed the backbone network, and the val_loss is above 3.5. The number of epochs is set to 400.
